includes/subscriptions/class-wc-payments-subscription-service.php

```diff
@@ -1004,4 +1004,26 @@ private function validate_subscription_data( $subscription_data ) {
 			}
 		}
 	}
+
+	/**
+	 * Determines if the store has any active WCPay subscriptions.
+	 *
+	 * @return bool True if store has active WCPay subscriptions, otherwise false.
+	 */
+	public static function store_has_active_wcpay_subscriptions() {
+		$results = wcs_get_subscriptions(
+			[
+				'subscriptions_per_page' => 1,
+				'fields'                 => 'ids',
+				'subscription_status'    => 'active',
+				// Ignoring phpcs warning, we need to search meta.
+				'meta_query'             => [ // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
```

Comment:
Is there a way to avoid the `meta_query` here?

Reply:
I don't think there's a way to avoid using meta_query here.

That said, this isn't the correct way to use the `'meta_query'` param.
It needs to be a nested inside an array for it to work.

For example:

```
$results = wcs_get_subscriptions(
	[
		'subscriptions_per_page' => 1,
		'subscription_status'    => 'active',
		'meta_query'             => [ // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
			[
				'key'     => '_wcpay_subscription_id',
				'compare' => 'EXISTS',
			]
		],
	]
);
```

Reply:
Just retested with this logic in place. I see now the flaw in my previous testing – I needed to test with WCS / tokenised subscriptions active. Since the `meta_query` wasn't doing anything I was getting false positives – warning shown when merchant only has active tokenised subs. Good catch 🏀

Reply:
I added a follow up issue to document `meta_query`: 

- https://github.com/Automattic/woocommerce-subscriptions-core/issues/337